src/Pages/Login/Login.js

```diff
@@ -0,0 +1,241 @@
+import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import styled from 'styled-components';
+import { KAKAO_LOGIN_API } from '../../config';
+
+// import './Login.scss';
+
+class Login extends Component {
+  kakaoLoginHandler = () => {
+    window.Kakao.Auth.login({
+      success: (auth) => {
+        fetch('http://10.58.3.58:8000/users/sign-up', {
```

Comment:
```suggestion
        fetch(`${API}/users/sign-up`, {
```
ip address 등은 자주 바뀔 수 있는 값이므로 src/config.js에 분리해두고 사용하면 좋을 것 같네요